assets/src/edit-story/app/story/effects/test/useLoadStory.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import useLoadStory from '../useLoadStory';
+import { createStory } from '../../../api/testUtils/story';
+import { useAPI, useHistory } from '../../../';
+
+jest.mock('../../../', () => ({
+  useHistory: jest.fn(),
+  useAPI: jest.fn(),
+}));
+
+function TestComponent({ restore }) {
+  useLoadStory({ storyId: 11, shouldLoad: true, restore });
+  return null;
+}
```

Comment:
To test a hook, please use `renderHook` from `@testing-library/react-hooks`. This function can also take another component as `wrapper` in order to [setup the required mock contexts](https://github.com/testing-library/react-hooks-testing-library/blob/master/docs/usage/advanced-hooks.md#context).

Reply:
Updated.